src/Core/Ripgrep.re

```diff
@@ -2,6 +2,24 @@ open Rench;
 
 module Time = Revery_Core.Time;
 
+// TODO: remove after 4.08
+module List = {
+  include List;
+
+  let filter_map = f => {
```

Comment:
We have a dup of this that we could replace / remove: https://github.com/onivim/oni2/blob/349fdbda43ada69c2de71c701a8b06202a2f936f/src/Core/Utility.re#L94

Reply:
I can move this into `Utility` like you've done with `Option` somewhere.